misc/deployer/build.gradle

```diff
@@ -0,0 +1,233 @@
+/**
+ * Automated Exploratory Tests
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.hidetake:gradle-ssh-plugin:2.2.0'
+    }
+}
+
+// In this section you declare where to find the dependencies of your project
+repositories {
+    // Use jcenter for resolving your dependencies.
+    // You can declare any Maven/Ivy/file repository here.
+    jcenter()
+}
+
+/*
+ * Set of tasks helpful to maintain and deploy bundles on Karaf
+ * collect all jars
+ * clearKaraf's data folder,
+ * stop,start,restart karaf,
+ * transfer jars via ssh to karafs deplo folder.
+ * creates zip package with jars.
+*/
+
+allprojects {
+
+    apply plugin: 'org.hidetake.ssh'
+
+    remotes {
+        vagrant {
+            role 'localhost'
+            host = 'localhost'
+            port = 2222
+            user = 'root'
+            password = "vagrant"
+            knownHosts = allowAnyHosts
+        }
+
+        int1 {
```

Comment:
IMO: no, public, opensource repository is for storing the code and generic configurations, not company's deployment parameters.